gwakeonlan/settings.py

```diff
@@ -43,6 +44,12 @@ def __init__(self):
     parser.add_option('-q', '--quiet', dest='verbose_level',
                       action='store_const', const=VERBOSE_LEVEL_QUIET,
                       help='hide error and information messages')
+    parser.add_option('-i', '--import-ethers-file', dest='ethers_file',
+                      help='Include the given ethers file')
```

Comment:
Surely that feature doesn't hurt but it won't be useful for a graphical application.
I should then add code for adding and removing hosts, as well as reading hosts from ARP cache and much more. That's out of scope for gWakeOnLAN, you still could acquire the same feature by clicking two buttons.